rbd/namespace_nautilus_test.go

```diff
@@ -73,4 +77,32 @@ func TestNamespace(t *testing.T) {
 		assert.NoError(t, err)
 		assert.Equal(t, val, false)
 	})
+
+	t.Run("NamespaceList", func(t *testing.T) {
+		var (
+			name1 = "name1"
+			name2 = "name2"
+			name3 = "name3"
+		)
+		err := NamespaceCreate(ioctx, name1)
+		assert.NoError(t, err)
+		err = NamespaceCreate(ioctx, name2)
+		assert.NoError(t, err)
+		err = NamespaceCreate(ioctx, name3)
+		assert.NoError(t, err)
+		defer func() {
+			assert.NoError(t, NamespaceRemove(ioctx, name1))
+			assert.NoError(t, NamespaceRemove(ioctx, name2))
+			assert.NoError(t, NamespaceRemove(ioctx, name3))
+		}()
+
+		eval, err := NamespaceExists(ioctx, name1)
+		assert.NoError(t, err)
+		assert.Equal(t, eval, true)
+		nList, err := NamespaceList(ioctx)
+		assert.NoError(t, err)
+		assert.Equal(t, nList[0], name1)
```

Comment:
I don't know if we want to rely on the order coming back always the same. Using a [Contains](https://godoc.org/github.com/stretchr/testify/assert#Contains) check or similar might be preferable.

Reply:
Tests in Ceph are doing the same, but changing it here for safer side.